scripts/ml/3_generate_embeddings.py

```
@@ -5,63 +5,106 @@
+        return 0
+
+    try:
+        cursor = conn.cursor()
+        # executemany is faster than individual execute calls
+        cursor.executemany(
+            "UPDATE content SET embedding = %s WHERE id = %s",
+            [(emb, cid) for cid, emb in ids_and_embeddings],
+        )
+        conn.commit()
+        return len(ids_and_embeddings)
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)